### ISSUE TITLE
_force_grafana_admin_password - metwork version 1.1 hardcoded

### ISSUE THREAD

**Issue by matthieumarrast** [metwork-framework/mfadmin#252]
In script https://github.com/metwork-framework/mfadmin/blob/master/adm/_force_grafana_admin_password.sh, 
the version metwork v1.1 is hardcoded:
`echo "${MFADMIN_GRAFANA_ADMIN_PASSWORD}" |/opt/metwork-mfadmin-1.1/opt/metrics/opt/grafana/bin/grafana-cli ...`

It raises error while starting grafana with metwork v2:
```
- Testing Grafana... ━━━━━━━━━━━━ [ RUNNING ] -:--:-- /opt/metwork-mfadmin-2.0/bin/_force_grafana_admin_password.sh: ligne 3: /opt/metwork-mfadmin-1.1/opt/metrics/opt/grafana/bin/grafana-cli: Aucun fichier ou dossier de ce type
- Testing Grafana...                                       [ ERROR ]
```
And we cannot connect to grafana with admin user.

**Comment by matthieumarrast** [comment]
As a workaround, execute below command in mfadmin environment:

`/opt/metwork-mfadmin-2.0/opt/metrics/opt/grafana/bin/grafana-cli --config "${MFMODULE_RUNTIME_HOME}/tmp/config_auto/grafana.ini" --homepath "${MFMODULE_HOME}/opt/metrics/opt/grafana" admin reset-admin-password --password-from-stdin`

Then seize the password for admin user. 
Afterward we can connect to grafana web client.

**Comment by matthieumarrast** [comment]
replace `/opt/metwork-mfadmin-1.1` by `${MFADMIN_HOME}`

**Comment by thebaptiste** [comment]
Closed by #253
